apps/openmw/mwrender/renderingmanager.cpp

```diff
@@ -222,6 +222,7 @@ namespace MWRender
         resourceSystem->getSceneManager()->setForceShaders(Settings::Manager::getBool("force shaders", "Shaders"));
```

Comment:
You're not force-enabling shaders here, so you're still going to end up with a bunch of objects using fixed-function mode.

Reply:
Actually, it looks like you're doing it elsewhere, so I guess what you've got would work. I'm somewhat inclined to say it's best to have it here as then it's guaranteed to be applied everywhere, even if something new is added by someone who doesn't know about it.